source/algorithms.tex

```diff
@@ -3390,7 +3390,7 @@
 \pnum
 \complexity
 For the overloads with no \tcode{ExecutionPolicy}, exactly
-\tcode{min((i - first) + 1, (last - first) - 1)}
+\[ \min(\tcode{(i - first) + 1}, \ \tcode{(last - first) - 1}) \]
```

Comment:
Are you sure we need the extra space after the comma? We don't generally do this at the moment (except once?); is it necessary?

Reply:
It looks much better that way; otherwise we have a wide space between "last - first" and a very small space after the comma.

Before:
![before](https://user-images.githubusercontent.com/23412755/49278116-1cbfea80-f484-11e8-8be8-61b1d73eb78a.png)

After:
![after](https://user-images.githubusercontent.com/23412755/49277998-ce125080-f483-11e8-9469-62dfd3b5d6f9.png)

Reply:
OK, fair enough, thanks!